<section id="jobs">
    <div class="container">
        <div class="row">
            <div class="col-lg-12 text-center">
                <h2 class="section-heading">Available Positions</h2>
                <hr class="primary">
                Beiko Lab in Dalhousie University (Halifax, Nova Scotia, Canada) is seeking 3 postdoctoral fellows:
            </div>
        </div>
    </div>

    <div class="container">
        <div class="row">

            <div class="col-lg-4 col-md-6 text-center">
            <div class="service-box">
            <h3>ARETE Postdoctoral Fellow #1 </h3>
            <h4>Position ID 2019-06</h4>
            <p class="text-muted", align="justify">
            Project will be building on existing tools (e.g., <a href="http://kiwi.cs.dal.ca/Software/RSPR">RSPR</a>) to infer phylogenetic trees and perform large-scale inference of LGT. Key challenges include robust, large-scale phylogenomic inference, and synthesizing different lines of evidence to build a comprehensive picture of gene transfer in large genomic datasets.
            This position requires someone with a background in either bioinformatics or computer science. <br />
            Key technical skills include: <br />
            <ul class='text-muted', align='left'>
                <li>Phylogenetics</li>
                <li>Microbial genomics and evolution</li>
                <li>Some programming (Python sufficient; C++ an asset)</li>
            </ul>
            </p>
            </div>
            </div>
            <div class="col-lg-4 col-md-6 text-center">
            <div class="service-box">
            <h3>ARETE Postdoctoral Fellow #2</h3>
            <h4>Position ID 2019-07</h4>
            <p class="text-muted", align="justify">
            Project involves developing methods to map inferred LGT events into an ecological context. This will include the association of events with specific habitat parameters (e.g., geographic location, habitat type, specific patterns of antibiotic usage), as well as “gene-ecology” methods that examine the distributional dynamics of microbial genes and map these into traditional ecological frameworks (e.g., <a href="https://academic.oup.com/femsre/article/38/1/90/510792">this work</a>). This position requires an individual with a background in bioinformatics and/or ecology. <br />
            Key technical skills include: <br />
            <ul class='text-muted' align='left'>
                <li> Microbial ecology </li>
                <li> Community ecology / metacommunity theory </li>
                <li> Data analysis skills in R/Python </li>
            </ul>
            </p>
            </div>
            </div>
            <div class="col-lg-4 col-md-6 text-center">
            <div class="service-box">
            <h3>AMRtime Postdoctoral Fellow</h3>
            <h4>Position ID 2019-08</h4>
            <p class="text-muted", align="justify">
            This fellow will work to extend existing work in the inference of antimicrobial-resistance genes from metagenomic data (e.g., <a href="https://github.com/beiko-lab/AMRtime">AMRtime</a>) using rapid homology filtering and search alongside machine-learning methods to assign metagenomic sequences to specific classes of AMR genes
The fellow will augment AMRtime with new models to better detect novel AMR variants that fall outside currently defined strict criteria for detection. Extending these models will require a combination of methods that encompass taxonomic assignment, sensitive homology search, mobile-element detection, and phylogenetics to differentiate legitimate AMR matches from homologs that do not confer a resistance phenotype.
            </p>
            </div>
            </div>
        </div>
        <p></p>
        <p class='text-muted'>
        All postdoctoral fellows will collaborate with other members of the project team, and be expected to give conference presentations and prepare journal articles based on their work. There will be opportunities to gain project-management experience through the ARETE project.</p>
        <p class="text-muted">
Please send a CV and statement of research interests to <a href="mailto:elvira.mitraka@gmail.com">Elvira Mitraka</a>. Applicants should indicate the position ID in the email subject header. Review of applications will begin on <strong>April 20, 2019</strong>. To receive full consideration, please submit your application by this deadline. Applications received after April 20 may also be considered.
        </p>
    </div>
</section>


